x/zkism/types/msgs_test.go

```diff
@@ -6,13 +6,12 @@ import (
 
 	"github.com/bcp-innovations/hyperlane-cosmos/util"
 	"github.com/celestiaorg/celestia-app/v6/x/zkism/types"
-	"github.com/celestiaorg/go-square/v2/share"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/stretchr/testify/require"
 )
 
-func TestMsgCreateZKExecutionISMValidateBasic(t *testing.T) {
-	var msg *types.MsgCreateZKExecutionISM
+func TestMsgCreateInterchainSecurityModuleValidateBasic(t *testing.T) {
```

Comment:
These test cases are failing now because of the removal of the code I mentioned in `msgs.go`